README.md

```diff
@@ -41,8 +41,23 @@ least one paver. Use `S` as the start state, and `r`, `g`, `b` as terminals that
 represent the tiles.
 
 ## A1.1
+S -> < empty >
 
-«replace this with your answer»
+S -> R S R
+
+S -> G S G
+
+S -> B S B
+
+S -> L
+
+R -> "r"
+
+G -> "g"
+
+B -> "b"
+
+L-> "r"|"g"|"b"
```

Comment:
Before I looked into this, I had to tidy it up a little. I removed the blank lines, and replaced the R,G,and B NTs with terminals:

~~~
S -> < empty >
S -> r S r
S -> g S g
S -> b S b
S -> L
L->  r|g|b
~~~

I then removed the L NT

~~~
S -> < empty >
S -> r S r
S -> g S g
S -> b S b
S -> r|g|b
~~~

And suddenly all is clear. And you're so, so close. The only problem is that very first line. Let's generate a production using `r S r`, and choose `S -> <empty> `. That gives us a result of `rr`, which is an even length string.

I suspect you did this deliberately, in order to allow both even length and odd length paths which are palidromic. But technically it is not what the question wanted.

# -1